voskcli/transcribe.py

```diff
@@ -79,6 +102,19 @@ def write_captions_paragraph(vtt, paragraph):
 
 
 def write_webvtt_captions(rec_results):
+    '''
+    Process transcription data.
+    
+    Split transcription data into lines of up to 35 chars and paragraphs
+    of up to 2 lines (based on MAX_CHARS_PER_LINE and MAX_LINES_IN_PARAGRAPH),
```

Comment:
I would usually keep this neutral, so that I don't need to update this if I ever change the value of those variables:

```suggestion
    Split transcription data into lines of up to MAX_CHARS_PER_LINE chars and paragraphs
    of up to MAX_LINES_IN_PARAGRAPH lines,
```